packages/main/src/plugin/kubernetes-context-state.ts

```diff
@@ -52,12 +53,20 @@ interface ContextState {
   resources: ContextStateResources;
 }
 
-// All resources managed by podman desktop
+// A selection of resources, to indicate the 'general' status of a context
+const selectedResources = ['pods', 'deployments'] as const;
```

Comment:
I think this is the first time someone protect its array in Podman Desktop codebase 🎉